engine/src/flutter/lib/web_ui/test/canvaskit/embedded_views_test.dart

```diff
@@ -596,7 +596,7 @@ void testMain() {
       skip: isSafari || isFirefox,
     );
 
-    test('removes the DOM node of an unrendered platform view', () async {
+    test('preserves the DOM node of an unrendered platform view', () async {
```

Comment:
LOL

Reply:
Lol, there's another test which checks that intermediate Picture objects are disposed by checking that a debug counter for "Pictures Created" equals "Pictures Disposed" which passes by virtue of both counters being `0` and never referenced outside of the test. I was going to delete it but I think since this is behavior we want to actually test I think I should just make it actually work and test the thing it's meant to test